code/go/internal/validator/semantic/validate_kibana_filter_present_test.go

```diff
@@ -64,6 +64,9 @@ func TestCheckDashboardHasFilter(t *testing.T) {
 			require.Len(t, files, 1, "looking for %s", c.dashboard)
 
 			err = checkDashboardHasFilter(files[0])
+			if err != nil {
+				t.Log(err)
+			}
```

Comment:
Is this a leftover or is it intended to help checking test output ?

Reply:
I found it useful when running the test with verbosity.